llabake/Recipes
# Ft/landing page 170235420

#### What does this PR do?
- Build the application landing page
 #### Description of Task to be completed?
- Build landing page
- Add background image
- Add search and top recipes section
 #### How should this be manually tested?
- Start the application by running `rails s`
- Go to `localhost/3000`
- The home page should be loaded
 #### What are the relevant pivotal tracker stories?
[Story](https://www.pivotaltracker.com/story/show/170235420)
 #### Any background context you want to add?
N/A
 #### Screenshots
<img width="1440" alt="Screenshot 2019-12-13 at 12 22 14 PM" src="https://user-images.githubusercontent.com/29184825/70796712-3be59d80-1da3-11ea-9376-76ed55b6c46d.png">
